loadflow/src/main/resources/config.yml

```diff
@@ -13,4 +13,8 @@ componentDefaultConfig:
 #  debug: true
 #  useSAX: false
   
-
+hades2:
+    homeDir: /home/tilloyann/simulators/hades2/hades2-V6.4.0.1.1/
+    timeout: 120
+    debug: true
+    useSAX: false
```

Comment:
Same comment here.

Reply:
Isn't there a config.yml file for the topology tutorial?

Reply:
Agree

Reply:
It is a mistake.